src/main/java/org/apache/sling/feature/cpconverter/handlers/SystemUsersEntryHandler.java

```diff
@@ -43,9 +43,10 @@ public void handle(String path, Archive archive, Entry entry, ContentPackage2Fea
             path = matcher.group(1);
         }
 
-        RepoPath currentPath = new RepoPath(PlatformNameFormat.getRepositoryPath(path)).getParent();
+        RepoPath originalPath = new RepoPath(PlatformNameFormat.getRepositoryPath(path));
+        RepoPath intermediatePath = originalPath.getParent();
```

Comment:
see above. that could also be move into the 'SystemUser' constructor.... but it depends a bit on the design and which class should be in charge of knowing this kind of details.